src/pages/authorization/authorizing-api-calls/index.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-weight: 30
+weight: 10
 category: "authorization"
 subcategory: "authorizing-api-calls"
 title: "Authorizing API calls"
```

Comment:
Some of the links in the **Use cases** table might no longer work as the documents were moved. For example, `[Personal Access Tokens](#personal-access-tokens)` will most likely not work.